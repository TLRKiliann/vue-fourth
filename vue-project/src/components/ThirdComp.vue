<script lang="ts">
import { defineComponent } from 'vue';
export default defineComponent({
  name: 'ThirdComp',
  inject: ['username'],
  data() {
    return {
    }
  },
})
</script>

<template>
    <h2>ThirdComp {{ username }}</h2>
</template>

<style scoped>

</style>