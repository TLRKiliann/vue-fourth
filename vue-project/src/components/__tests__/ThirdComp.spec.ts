import { describe, it, expect } from 'vitest'

import { mount } from '@vue/test-utils'
import ThirdComp from '../ThirdComp.vue'

describe('ThirdComp', () => {
  it('snapshot UI testing', () => {
    const wrapper = mount(ThirdComp,{});
    expect(wrapper.text()).toMatchSnapshot()
  })
})