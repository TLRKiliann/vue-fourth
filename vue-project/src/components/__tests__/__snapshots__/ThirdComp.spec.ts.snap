// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ThirdComp > snapshot UI testing 1`] = `"ThirdComp"`;
